interfaces/apparmor/spec.go

```diff
@@ -42,7 +42,7 @@ type Specification struct {
 	snippets map[string][]string
 	// updateNS describe parts of apparmor policy for snap-update-ns executing
 	// on behalf of a given snap.
-	updateNS []string
+	updateNS strutil.OrderedSet
```

Comment:
why does this need to be an _ordered_ set? We only seem to use the ordering in the tests, and even there it seems suspiciously testing just that the order is what is expected, not necessarily some larger bit which requires it to be ordered.

Reply:
If we ever want to audit the text we produce I would argue that we want to have this property.

Reply:
okay that's fair